content/spin/v2/contributing-spin.md

```diff
@@ -9,11 +9,31 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/contributi
 - [Before You Commit](#before-you-commit)
 - [Committing and Pushing Your Changes](#committing-and-pushing-your-changes)
 
-We are delighted that you are interested in making Spin better! Thank you! This
-document will guide you through making your first contribution to the project.
+We are delighted that you are interested in making Spin better! Thank you!
+
+This document will guide you through making your first contribution to the project.
 We welcome and appreciate contributions of all types — opening issues, fixing
 typos, adding examples, one-liner code fixes, tests, or complete features.
 
+
+## Developer Community Calls
+
+<div style="display: block" align="left">
```

Comment:
In the preview, this was very slow to appear - the "recent contributors" appeared to be empty for several seconds.  Is this expected?  Is it okay?

Reply:
The contributor widget is slow loading latest data via ossinsights.io. If it feels weird I can take the widget out, it's a nice-to-have